src/defaultHost.ts

```diff
@@ -0,0 +1,2 @@
+export const defaulPort = '11434';
+export const defaultHost = `http://127.0.0.1:${defaulPort}`;
```

Comment:
better you can give generic name `constant.js` that store all the constant here of `defaultHost.js`

There is wrong spelling too.
```suggestion
export const port = '11434';
export const host = `http://127.0.0.1:${port}`;
```